ui-v2/app/routes/dc/acls/tokens/index.js

```diff
@@ -0,0 +1,29 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
+import { get } from '@ember/object';
+import WithTokenActions from 'consul-ui/mixins/token/with-actions';
+export default Route.extend(WithTokenActions, {
+  repo: service('tokens'),
+  settings: service('settings'),
+  queryParams: {
+    s: {
+      as: 'filter',
+      replace: true,
+    },
+  },
+  model: function(params) {
+    const repo = get(this, 'repo');
+    return hash({
+      ...repo.status({
+        items: repo.findAllByDatacenter(this.modelFor('dc').dc.Name),
+      }),
```

Comment:
Here's one place where `repo.status()` is being used. The idea is it doesn't move too far away from what I was doing before. The `status` method basically catches any HTTP errors (in this case 401 and 403) and then resolves with extra properties on the `{item: ...}` so I know whether ACLs are enabled or whether you are authorised.

The `repo` seemed like the best place to do this, still about 80% on that decision though.

Reply:
Looking at the new API missing things give me 403's, but yeah good call, will check this.

Reply:
Just tried this, changed it so 404's (which are actually 403's) to give me the ACL login, thanks for that!